src/legacy/core_plugins/vega/public/plugin.ts

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
I would prefer we put shims in a dedicated folder. So we know only that only NP code is in that folder. Similar to how other plugins do that, for example, `dashboard_embeddable_container` or `spaces`, or `apm`.

So, for example, this file would be in 

```
src/legacy/core_plugins/vega/public/shim/public/plugin.ts
```

instead.

I'm just adding this as a note here. @lukeelmers maybe we can discuss it on Tuesday.

Reply:
@streamich 👍  sgtm. we can discuss and follow up here.